### PR TITLE
add material-ui table component example

### PR DESCRIPTION
**Pull request by Julian-B90**

### CODE REVIEW

**Comment by Julian-B90** [comment]
For the Sandbox example i get this error `Only a max of 250 modules is allowed in a sandbox`

**Comment by Julian-B90** [comment]
Codesandbox example:
https://codesandbox.io/s/github/weluse/react-table/tree/master/examples/material-UI-components

File: examples/material-UI-components/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and Rescripts.
+
+You can:
+
+- [Open this example in a new CodeSandbox](https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/basic)
```
Comment (tannerlinsley) [inline examples/material-UI-components/README.md]: Let's get this link updated

File: examples/material-UI-components/src/App.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                    <TableCell {...cell.getCellProps()}>
+                      {cell.render('Cell')}
+                    </TableCell>
+                  )
+                })}
+              </TableRow>
+            )
+        )}
+      </TableBody>
+    </MaUTable>
```
Comment (tannerlinsley) [inline examples/material-UI-components/src/App.js]: Fantastic!

**Comment by tannerlinsley** [comment]
I think all we need now is to update the docs!

https://github.com/tannerlinsley/react-table/blob/master/docs/examples.md

File: examples/material-UI-components/README.md (hunk 1 above)
Comment (Julian-B90) [reply examples/material-UI-components/README.md]: I have update the link.

**Comment by Julian-B90** [comment]
@tannerlinsley i have update the docs :-)

**Comment by tannerlinsley** [comment]
Looks great!

**Comment by MareoRaft** [comment]
@Julian-B90 Looks good to me.  Do you happen to have an example that includes pagination?

**Comment by Julian-B90** [comment]
@MareoRaft her is an example with simple pagination https://codesandbox.io/s/github/weluse/react-table/tree/example-with-pagination/examples/material-UI-components

**Comment by sureshtidbit** [comment]
An example is not working. @Julian-B90

**Comment by Julian-B90** [comment]
@sureshtidbit the codesandbox have an error but you can use this https://github.com/weluse/react-table/tree/example-with-pagination/examples/material-UI-components

**Pull request merged**